projects/ondofinance/index.js

```diff
@@ -61,18 +61,29 @@ async function addEthBalances(addresses, block, balances) {
 
 async function tvl(timestamp, block, chainBlocks) {
     const balances = {};
+    let partner_tokens = STABLE_PARTNER_TOKENS
+    let ondo_multisigs = STABLE_PARTNER_VAULTS
+    let ondo_lps = PARTNER_LPS
+    try {
+        const data = await (await fetch("https://data.ondo.finance/v1/addresses")).json()
+        partner_tokens = data["supported_tokens"]
+        ondo_multisigs = data["ondo_multisigs"]
+        ondo_lps = data["ondo_lps"]
+    } catch {
+        console.log("Failed to data from Ondo Finance, resorting to defaults")
```

Comment:
Hi, the change looks good. But we would like to know when there is an issue, and much prefer you throwing an error and failing. Another reason is, now the chart would be inconsistent, api fail leads to lower tvl right?

Reply:
Fair, in that case, the try catch is probably unnecessary all together, will go ahead and remove.